stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -34,12 +40,16 @@ public class AttachmentGallery : ConstraintLayout {
 
     public fun init(attr: AttributeSet?) {
         attr?.let(::configureAttributes)
+        configureImagesMenu()
     }
 
     public fun provideImageList(fragmentActivity: FragmentActivity, imageList: List<String>, currentIndex: Int = 0) {
         binding.attachmentGallery.adapter = AttachmentSlidePagerAdapter(fragmentActivity, imageList)
         configPositionCount(imageList.size)
         binding.attachmentGallery.setCurrentItem(currentIndex, false)
+
+        this.imageList.clear()
+        this.imageList.addAll(imageList)
```

Comment:
Perhaps having a `var` `List` would be simpler here? It would also ensure we're only modifying the entire thing at the same time.